StudCity.Frontend/src/pages/authenticate/Authenticate.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import Label from "../../UI/Label";
+import TextField from "../../UI/Fields/TextField";
+import PasswordTextField from "../../UI/Fields/PasswordTextField";
+import Button from "../../UI/Button";
+import GoogleButton from "../../UI/GoogleButton";
+function Authenticate() {
+	const [disabled, setDisabled] = useState(false)
+
+	return (
+		<div className="w-1/1 h-screen flex">
+			<div className="form w-1/2 h-screen flex bg-[#f9fcf8]">
+				<div className="container flex flex-col w-4/5 m-auto h-[85%] bg-[#F9FCF8] rounded-3xl p-11 shadow-md ">
+					<h1 className="w-full text-center font-bold text-4xl my-3 text-[#486645]">Welcome back</h1>
+					<h6 className="w-full text-center font-normal text-lg text-[#486645]">Please enter your details</h6>
+					<form  className="flex flex-col w-1/1">
+						<Label>Email</Label>
+						<TextField 
+							placeholder = {"Enter your email"} 
+							className = {""} type = {"email"} 
```

Comment:
add a default value for className prop of TextField and other components where needed